modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -129,8 +129,14 @@ protected DDMFormField getDDMFormField(JSONObject jsonObject) {
 			jsonObject.getJSONArray("options"), ddmFormField);
 		setDDMFormFieldPlaceholder(
 			jsonObject.getString("placeholder"), ddmFormField);
+		setDDMFormFieldProperty(
+			ddmFormField, "buttonLabel", jsonObject.getString("buttonLabel"));
+		setDDMFormFieldProperty(
+			ddmFormField, "title", jsonObject.getString("title"));
 		setDDMFormFieldPropertyDDMStructureId(jsonObject, ddmFormField);
 		setDDMFormFieldPropertyDDMStructureLayoutId(jsonObject, ddmFormField);
+		setDDMFormFieldPropertyMessage(
+			ddmFormField, jsonObject.getString("message"));
```

Comment:
Considering the [change](https://github.com/natocesarrego/liferay-portal/pull/1123/commits/060da3f7453a1f3161646321c729e9ac5132f3c2) on the first commit, shouldn't this code refer to `messageKey` instead of `message`?

Reply:
Hey, @natocesarrego. The `messageKey` is not passed on to the frontend, only the `message` property that's created in the Redirect Button contributor. So, since this is the template context processor, the data that comes from the frontend is just the `message`. Let me know if you have further questions. 😄